msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -3,15 +3,17 @@
   <Fragment>
     <UI>
       <Dialog Id="GitChooseEnvironmentDlg" Width="370" Height="270" Title="!(loc.GitChooseEnvironmentDlg_Title)" X="50" Y="50">
-        <Control Type="RadioButtonGroup" Property="CLIENVIRONMENT" Id="RadioButtonGroup" Width="310" Height="55" X="30" Y="87">
-          <RadioButtonGroup Property="CLIENVIRONMENT">
+        <Control Type="RadioButtonGroup" Property="UI_CliEnvironment" Id="RadioButtonGroup" Width="310" Height="55" X="30" Y="87">
+          <RadioButtonGroup Property="UI_CliEnvironment">
```

Comment:
On Wed, 11 May 2016, Craig E. Shea wrote:

> > @@ -3,15 +3,17 @@
> >    <Fragment>
> >      <UI>
> >        <Dialog Id="GitChooseEnvironmentDlg" Width="370" Height="270" Title="!(loc.GitChooseEnvironmentDlg_Title)" X="50" Y="50">
> > -        <Control Type="RadioButtonGroup" Property="CLIENVIRONMENT" Id="RadioButtonGroup" Width="310" Height="55" X="30" Y="87">
> > -          <RadioButtonGroup Property="CLIENVIRONMENT">
> > +        <Control Type="RadioButtonGroup" Property="UI_CliEnvironment" Id="RadioButtonGroup" Width="310" Height="55" X="30" Y="87">
> > +          <RadioButtonGroup Property="UI_CliEnvironment">
> 
> Sometimes the journey is more important than the destination. Also, I misunderstood this area of the original installer anyway, which resulted in some things being wrong from the outset, and this commit reflects new knowledge via these corrections. Lastly, I changed the names of the radio buttons and their values, and that was a mistake. It'll be much easier for us to relate the MSI radio buttons to their InnoSetup counterparts if the "terminology" is kept the same, and that was the main impetus for the changes in the UI--consistency.

I agree that the journey is an important part of the adventure.

However, the commit history is more useful if it does not really reflect
all the wrong turns, except by way of relating in commit messages why
those turns were wrong if it is not obvious.

In other words, a nicely rewritten commit history tells a nice story in
and of itself, and it does not necessarily reflect the entire development.
Makes for much more pleasant reading, though. And for *substantially*
nicer bisecting, should it be needed.